buildspec.yml

```diff
@@ -0,0 +1,35 @@
+version: 0.1
+
+environment_variables:
+  plaintext:
+    DATE_FORMAT: "%F %H:%M:%S"
+
+phases:
+  install:
+    commands:
+      - echo "$(date +"${DATE_FORMAT}") - Begin install phase (pre-build)"
+      - echo "Install jscs..."
+      - npm install -g jscs
+  pre_build:
+    commands:
+      - echo "$(date +"${DATE_FORMAT}") - Start build"
+      - echo "AWS CodeBuild src directory: ${CODEBUILD_SRC_DIR}"
```

Comment:
@johncrossman In my local YAML parser, this line is getting parsed as a hash rather than a string"

{code}
{"echo \"AWS CodeBuild src directory"=>"${CODEBUILD_SRC_DIR}\""}
{code}

Because of the presence of the colon, I think the entire string needs to be quoted (and perhaps we should enclose all command string in single quotes just to avoid ambiguity).